docs/api/ShallowWrapper/childAt.md

```diff
@@ -0,0 +1,28 @@
+# `.childAt(index) => ShallowWrapper`
+
+Returns a new wrapper with child at the specified index.
+
+#### Arguments
+
+1. `index` (`number`): A zero-based integer indicating which node to retrieve.
+
+
+#### Returns
+
+`ShallowWrapper`: A new wrapper that wraps the resulting node.
+
+
+
+#### Examples
+
+```jsx
+const wrapper = shallow(<ToDoList items={items} />);
+expect(wrapper.find('ul').childAt(0).type).to.equal('li');
```

Comment:
just to double check, by `s/type/type()` you mean that I should write: `wrapper.find('ul').childAt(0).type()` correct? 

I forgot that type is a method...